### PR TITLE
chore(dashboard): a few more fixes

### PR DESCRIPTION
**Pull request by LetItRock**
### What changed? Why was the change needed?

A few more small improvements.

### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: apps/dashboard/src/components/workflow-editor/steps/configure-step-content.tsx [hunk 1]
```diff
@@ -65,15 +66,19 @@ export const ConfigureStepContent = () => {
...
-      <SidebarContent>
-        <Link to={'./edit'} relative="path">
-          <Button variant="outline" className="flex w-full justify-start gap-1.5 text-xs font-medium" type="button">
-            <RiPencilRuler2Fill className="h-4 w-4 text-neutral-600" />
-            Configure {step?.type} template <RiArrowRightSLine className="ml-auto h-4 w-4 text-neutral-600" />
-          </Button>
-        </Link>
-      </SidebarContent>
-      <Separator />
+      {!EXCLUDED_EDITOR_TYPES.includes(step?.type ?? '') && (
```
Comment (LetItRock) [inline apps/dashboard/src/components/workflow-editor/steps/configure-step-content.tsx:69]: do not show the `Configure {channel} template` button for the step types: digest, delay, custom, trigger

File: apps/dashboard/src/components/workflow-editor/steps/edit-step-sidebar.tsx [hunk 2]
```diff
@@ -34,9 +35,21 @@ export const EditStepSidebar = () => {
   );
 
   const handleCloseSidebar = () => {
-    navigate('../', { relative: 'path' });
+    navigate('..', { relative: 'path' });
```
Comment (LetItRock) [inline apps/dashboard/src/components/workflow-editor/steps/edit-step-sidebar.tsx:38]: removed `/` at the end of the URL

File: apps/dashboard/src/components/workflow-editor/steps/edit-step-sidebar.tsx [hunk 3]
```diff
@@ -34,9 +35,21 @@ export const EditStepSidebar = () => {
...
+
+  useEffect(() => {
+    if (isNotSupportedEditorType) {
+      navigate('..', { relative: 'path' });
+    }
+  }, [isNotSupportedEditorType, navigate]);
+
+  if (isNotSupportedEditorType) {
+    return null;
+  }
```
Comment (LetItRock) [inline apps/dashboard/src/components/workflow-editor/steps/edit-step-sidebar.tsx:51]: if step type is not supported, we should not allow opening the editor

File: apps/dashboard/src/components/workflow-editor/steps/in-app/in-app-tabs.tsx [hunk 4]
```diff
@@ -34,7 +34,7 @@ import {
   AlertDialogTitle,
 } from '@/components/primitives/alert-dialog';
 
-const tabsContentClassName = 'h-full w-full px-3 py-3.5';
+const tabsContentClassName = 'h-full w-full px-3 py-3.5 overflow-y-auto';
```
Comment (LetItRock) [inline apps/dashboard/src/components/workflow-editor/steps/in-app/in-app-tabs.tsx:37]: scrollbar for the In-App Editor drawer content when there is not enough space

**Review by SokratisVidros — APPROVED** [review]

**Pull request merged**